File: server/Diskollection/Diskollection/Models/Artist.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Diagnostics.CodeAnalysis;

namespace Diskollection.Models
{
    public class Artist
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Country { get; set; }
        public Boolean IsActive { get; set; }
        public string Biography { get; set; }
        [AllowNull]
        public List<Album>? Albums { get; set; }
    }
}

File: client/Diskollection/src/app/shared/models/Artist.ts
import { Album } from "./Album";

export interface Artist{
    id: string;
    name: string;
    country: string;
    isActive: boolean;
    biography?: string;
    albums: Array<Album>;
}

export interface ArtistDTO{
    id: string;
    country: string;
    name: string;
    biography?: string; 
    isActive: boolean;
}